pkg/types/openstack/platform.go

```diff
@@ -3,7 +3,8 @@ package openstack
 // Platform stores all the global configuration that all
 // machinesets use.
 type Platform struct {
-	// Region specifies the OpenStack region where the cluster will be created.
+	// Region
```

Comment:
can we keep the field comment on the region.. i don't see the reason of removing it..

Reply:
also update the https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md

Reply:
Ack. Will be done